src/main/java/frc/robot/trajectory/Point.java

```diff
@@ -0,0 +1,45 @@
+package frc.robot.trajectory;
+
+public class Point {
```

Comment:
Consider using the Translation2D class provided by WPILib https://first.wpi.edu/wpilib/allwpilib/docs/release/java/edu/wpi/first/math/geometry/Translation2d.html